packages/atomic/src/pages/examples/commerce-website/listing-surf-accessories.html

```diff
@@ -25,7 +26,88 @@ <h1>Product listing page</h1>
     <main>
       <section>
         <atomic-commerce-interface type="product-listing">
-          <atomic-commerce-product-list> </atomic-commerce-product-list>
+          <atomic-commerce-product-list display="grid" density="compact" image-size="small">
+            <atomic-product-template>
+              <template>
+                <style>
```

Comment:
Can we remove this style tag for the time being ? Keep it as light as possible, and polish it up later when things are more stable.

A lot of those might change with the product components we will add.

 For example, for product image, there will be a dedicated component just for this, which will embed the styling done here.